components-sdk/src/locales/es.json

```
@@ -0,0 +1,63 @@
+{
+  "components.content": "Texto",
```

```suggestion
  "components.content": "Contenido",
```

Even if this is not inline with the API, it's more inline with the rest of the translations

---

I asked in the Discord server for elaboration about why Content was used, I think it'll be better to wait for opinions on that